src/applications/vaos/actions/newAppointment.js

```diff
@@ -176,12 +176,11 @@ export function updateFacilityType(facilityType) {
   };
 }
 
-export function startDirectScheduleFlow(appointments) {
+export function startDirectScheduleFlow() {
   recordEvent({ event: 'vaos-direct-path-started' });
 
   return {
     type: START_DIRECT_SCHEDULE_FLOW,
-    appointments,
```

Comment:
These past appointments get sent to the reducer during the eligibility check instead.